.github/workflows/backport.yml

```diff
@@ -0,0 +1,20 @@
+name: Backport
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  backport:
+    name: Create backport PRs
+    if: github.event.pull_request.merged == true
```

Comment:
shouldn't we also check to see if it has the `needs: port to stable` tag? Otherwise this looks like it will backport everything.

Reply:
I think the backport action handles that

Reply:
Yeah, backport action handles label checking.

Reply:
ah, okay.

Reply:
I guess I should read the original post more :)